en/06_Project_Governance/08_Public_API.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Definition of Public API
+summary: A definition of what we consider to be "public API" which is protected by semantic versioning
+icon: code
+---
+
+# Definition of public API
+
+The following is a definition of what we consider to be "public API". Public API is protected by semantic versioning.
+
+New public API will not be introduced in patches, but can be introduced in minor and major releases.
+
+Existing public API will not be removed or altered in patches or minor releases, but can be removed or altered in major releases.
+
+## Explicitly included
+
+These are explicitly included in our definition of public API (unless excluded below):
+
+* **global** functions, constants, and variables
+* namespaces, classes, interfaces, enums, and traits
+* public and protected scope (including methods, properties and constants)
+* private static class properties (considered to be configuration variables)
+* yml configuration file structure and value types
```

Comment:
```suggestion
* yml configuration file structure and value types
```
what is "yml configuration file structure"? And what is a yml "value type"?

Reply:
That was existing wording... I've done my best to clarify it now to what I _think_ was intended.